utils/auto_dockerfile_update/update_dockerfiles.py

```diff
@@ -1,19 +1,19 @@
 import os
-
 import argparse
-
-from get_dockerfiles import get_docker_files
+from get_dockerfiles import get_docker_files,get_file_path_and_docker_version_if_exist
 from get_latest_tag import get_latest_tag, parse_versions
-from typing import Dict, List
+from typing import Dict, List,Tuple
 import dateutil
 from git import Repo, GitCommandError
 import re
 from get_dockerfiles import LAST_MODIFIED_REGEX
 from datetime import datetime, timezone
 from functools import reduce
+import subprocess
 
 BATCH_SIZE = 1
-
+PYTHON_VERSION_PIPFILE_REGEX = re.compile(r"python_version = \"([^\"]+)\"")
```

Comment:
lets put Pipfile and Pyproject as the first words in these consts. Think thats easier to read